sonic-xcvrd/scripts/xcvrd

```diff
@@ -440,6 +462,13 @@ def main():
             # TODO: next step need to define more error types to handle accordingly.
             break
 
+        time_now = time.time()
+        time_diff = time_now - time_last_recovery_run
+        # time.time() returns value in unit of seconds while XCVRD_MAIN_THREAD_SLEEP_MSECS is defined in unit of milliseconds
```

Comment:
please remove unused code

Reply:
Hi Liat, this code is not unused.
this part of code is backported from master which has an independent thread to recover_missing_sfp_table_entries periodically. 
In 201811 we don't have a thread to do that and have to make use of the main thread. to avoid consuming too much cpu time this function shouldn't be called too frequently. currently it is called every minute, which is implemented by adding a timeout argument to get_transceiver_change_event.
However, in case of multiple SFP modules inserted/removed, get_transceiver_change_event can be woken up more frequently thus recover_missing_sfp_table_entries being called several times in a short period.
To avoid that, we implement a time_diff calculation.